### PR TITLE
hack: Add $GOPATH to hack/update-generated-xxx.sh scripts

### PR DESCRIPTION
**Pull request by stefwalter**
So that not everyone has to figure this out.

### CODE REVIEW

File: hack/update-generated-deep-copies.sh [hunk 1]
```diff
@@ -40,7 +45,7 @@ package ${version}
 // AUTO-GENERATED FUNCTIONS START HERE
 EOF
 
-	GOPATH=$(godep path):$GOPATH go run cmd/gendeepcopy/deep_copy.go -v ${version} -f - -o "${version}=" >>  $TMPFILE
+	GOPATH=$(godep path):${GOPATH:=} go run cmd/gendeepcopy/deep_copy.go -v ${version} -f - -o "${version}=" >>  $TMPFILE
```
Comment (wojtek-t) [inline hack/update-generated-deep-copies.sh]: Can you please change the above script so that it's exactly the same?

Comment (stefwalter) [inline hack/update-generated-deep-copies.sh]: Removed this erroneous change. The others should be enough.

**Comment by wojtek-t** [comment]
LGTM

**Comment by wojtek-t** [comment]
cc @davidopp for lead LGTM

**Comment by wojtek-t** [comment]
jenkins failure is unrelated to these changes:
Test Result (1 failure / -4)
Kubernetes e2e suite.Addon update should propagate add-on file changes

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by wojtek-t** [comment]
Cluster didn't started.

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by davidopp** [comment]
LGTM

Jenkins failures are not related to this PR. I'll re-run anyway but this is OK to merge.

**Comment by davidopp** [comment]
@k8s-bot test this please

**Pull request merged**